pkg/apis/extensions/v1alpha1/deepcopy_generated.go

```diff
@@ -1,4 +1,3 @@
-//go:build !ignore_autogenerated
```

Comment:
I think you need to update Go to 1.17.

Reply:
I am on 1.17!

Reply:
Hm, strange that this got removed, because Go 1.17 is exactly what introduced that line in the first place.
No big deal, I can double check and run codegen myself before we merge to ensure we don't "downgrade" files.